### PR TITLE
Add recipe for consult-flyspell

### PR DESCRIPTION
**Pull request by OlMon**
### Brief summary of what the package does

Integration of flyspell into consult.

### Direct link to the package repository

https://gitlab.com/OlMon/consult-flyspell

### Your association with the package

I am the maintainer/author.

### Relevant communications with the upstream package maintainer

None needed

### Checklist

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] `M-x checkdoc` is happy with my docstrings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [ ] I have confirmed some of these without doing them

### CODE REVIEW

File: recipes/consult-flyspell [hunk 1]
```diff
@@ -0,0 +1,4 @@
+(consult-flyspell
+ :repo "OlMon/consult-flyspell"
+ :fetcher gitlab
+ :files ("consult-flyspell.el"))
```
Comment (riscy) [inline recipes/consult-flyspell:4]: Equivalent and gives your package room to grow:
```suggestion
(consult-flyspell
 :repo "OlMon/consult-flyspell"
 :fetcher gitlab)
```

**Review by riscy — COMMENTED** [review]
Thanks for this!  Very quick first pass --

`consult-flyspell.el` with byte-compile using Emacs 27.1:
```
consult-flyspell.el:113:1:Warning: the function `word-at-point' is not known
    to be defined.
```
A depency on `thing-at-point` needs to be made, I believe.

`consult-flyspell.el` with package-lint 20220213.1457:
```
1 issue found:
8:55: error: Package flyspell is not installable.
```

`consult-flyspell.el` with [melpazoid](https://github.com/riscy/melpazoid):
- consult-flyspell.el#L77: Can the `format` and `concat` be combined?

Package and license:
- Add a LICENSE file to the repository

**Comment by OlMon** [comment]
Hej Hej,

I am sorry for the mistakes, some should be clear from the start... I fixed everything now.

Cheers!

**Comment by riscy** [comment]
Thank you!  All set here, and welcome to MELPA. :)

**Pull request merged**